src/FlightMap/MapItems/CameraTriggerIndicator.qml

```diff
@@ -29,13 +29,52 @@ MapQuickItem {
 
         readonly property real _radius: ScreenTools.defaultFontPixelHeight * 0.6
 
-        QGCColoredImage {
-            anchors.margins:    3
-            anchors.fill:       parent
-            color:              "white"
-            mipmap:             true
-            fillMode:           Image.PreserveAspectFit
-            source:             "/qmlimages/camera.svg"
+        // Old icon, disabled due to performance issues when too much icons in the map 
+        
+        // QGCColoredImage {
+        //     anchors.margins:    3
+        //     anchors.fill:       parent
+        //     color:              "white"
+        //     mipmap:             true
+        //     fillMode:           Image.PreserveAspectFit
+        //     source:             "/qmlimages/camera.svg"
+        // }
```

Comment:
Why is this code here ? It should be just removed.

Reply:
As this was a provisional patch until we sort out  how to do it with openGL shaders I thought it would be nice to leave that commented out. It is removed now, thanks for commenting.